style.css

```diff
@@ -61,19 +81,99 @@ h2 {
   background: rgb(156, 156, 156);
 }
 
-.each-book {
-  display: flex;
-  justify-content: space-between;
-  align-items: center;
-}
-
 .book-title {
   font-family: "lato", sans-serif;
   font-size: 14px;
 }
 
 .remove-button {
   width: fit-content;
-  padding: 6px;
-  border: 5px solid black;
+  height: 40px;
+  border: 3px solid black;
+}
+
+.remove-button:hover {
+  background-color: rgb(223, 221, 221);
+}
+
+.remove-button:active {
+  background-color: white;
+}
+
+/* Add Book Section */
+.add-book {
+  display: none;
+  flex-direction: column;
+  align-items: center;
+}
+
+#form {
+  display: flex;
+  flex-direction: column;
+  gap: 0.5rem;
+}
+
+.form-input {
+  border: 4px solid black;
+  padding: 0.5rem;
+}
+
+#submit {
+  padding: 0.5rem 3rem;
+  font-weight: bold;
+  align-self: end;
+  border: 4px solid black;
+  background-color: white;
+}
+
+#submit:hover {
+  background: rgb(180, 176, 176);
+  color: white;
+}
+
+#submit:active {
+  background: rgb(121, 119, 119);
+  color: white;
+}
+
+/* Contact info */
+.contact-info {
+  margin: auto;
+  width: 70%;
+  display: none;
+  flex-direction: column;
+}
+
+footer {
+  border: 2px solid black;
+  padding: 5px 20px;
+  margin-top: 100px;
+}
```

Comment:
- [ ] [REQUIRED] Kindly ensure the footer stays at the end of the page/beneath as it is almost at the center of the page currently. This will make the project look more professional.

### Your Implementation
![Screenshot from 2022-03-16 23-43-35](https://user-images.githubusercontent.com/58692590/158703933-38150a70-a25b-41ab-a2e7-ca0ea152ef5e.png)